### DISCUSSION TITLE
How to variabilise ENV values cleanly?

### DISCUSSION THREAD

**Discussion by rgarrigue** [community/community#25479] [Actions]
<p>Hi there,</p>
<p>I’ve exactly the same question as in this issue : <a href="https://github.com/rtCamp/action-slack-notify/issues/48" rel="noopener nofollow ugc">https://github.com/rtCamp/action-slack-notify/issues/48</a></p>
<p>This works :</p>
<pre><code class="lang-auto">    - name: Slack Notification - Success
      if: success()
      uses: rtCamp/action-slack-notify@v2.0.0
      env:
        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}

    - name: Slack Notification - Failure
      if: cancelled() || failure()
      uses: rtCamp/action-slack-notify@v2.0.0
      env:
        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
        SLACK_COLOR: 'danger'
</code></pre>
<p>But isn’t it possible to do something smarter, a bit like this maybe ?</p>
<pre><code class="lang-yaml"># Wild guess, don't copy paste
    - name: Slack Notification
      uses: rtCamp/action-slack-notify@v2.0.0
      env:
        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
        SLACK_COLOR: ${{ success() ? 'good' : 'danger' }}
</code></pre>
<p>Thanks,</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/rgarrigue">@rgarrigue</a>,</p>
<p>You can try like as this mentioned by <a class="mention" href="/simran-b">@Simran-B</a> <a href="https://github.community/t/possible-to-use-conditional-in-the-env-section-of-a-job/135170/2">here</a>:</p>
<pre><code class="lang-yaml">env:
  SLACK_COLOR: "${{ job.status == 'success' &amp;&amp; 'good' || 'danger' }}"
</code></pre>
